include/httpp/utils/SortedVectorKP.hpp

```diff
@@ -31,7 +31,13 @@ struct case_insensitive
         }
         else if (left.size() == right.size())
         {
+#ifdef WIN32
+# define strncasecmp(x, y, z) _strnicmp(x, y, z)
```

Comment:
@daedric what about introducing a utils header which includes this define or is it ok to have it multiple times all over the place?